h2o-py/tests/testdir_algos/grid/pyunit_grid_manual_export_resume_new_hyperspace.py

```diff
@@ -15,12 +15,13 @@ def grid_resume():
     # Run GBM Grid Search
     ntrees_opts = [1, 3]
     learn_rate_opts = [0.1, .05]
-    hyper_parameters = OrderedDict()
-    hyper_parameters["learn_rate"] = learn_rate_opts
-    hyper_parameters["ntrees"] = ntrees_opts
+    hyper_parameters = {
+        "learn_rate": learn_rate_opts,
+        "ntrees": ntrees_opts
+    }
     print("GBM grid with the following hyper_parameters:", hyper_parameters)
     
-    export_dir = pyunit_utils.locate("results")
```

Comment:
Why this change? It is better to output file to results directory, then they are preserved if anything fails.

Reply:
I thought only test results are to be written to the results dir, tmp data should be in tempfiles, same for junit

Reply:
That depends on what is tmp data to you but this use-case is imho similar to eg. checkpointing and those tests use pyunit_utils.locate("results"), example https://github.com/h2oai/h2o-3/blob/master/h2o-py/tests/testdir_algos/xgboost/pyunit_checkpoint_xgboost.py#L30

For data exchange for tests we follow the pyunit_utils.locate("results") method because it has a clear lifecycle.

The temp file will be gone with the docker exiting on Jenkins which could be a bad thing for reproducing a bug.